appservice/src/SiteWrapper.ts

```diff
@@ -158,12 +171,11 @@ export class SiteWrapper {
     }
 
     public async isHttpLogsEnabled(client: WebSiteManagementClient): Promise<boolean> {
```

Comment:
Where is this called?  Does it return the SiteLogsConfig now just to verify that the logs were updated?

Reply:
It's called in the app service extension. I mainly did it so we're consistent with client.webApps.updateDiagnosticLogsConfigSlot (which returns SiteLogsConfig)